internal/stringutil/find.go

```diff
@@ -0,0 +1,39 @@
+package stringutil
+
+// FindQuoted returns the indexes of the instance of v in s, or empty slice if v is not present in s.
+// It ignores v present inside quoted runs.
+func FindQuoted(s string, v rune, quote rune) []int {
```

Comment:
If I am understanding it's operation correctly, should this be called FindUnquoted?   Similar comment on the `quotedIndices` var name.

Reply:
That's true, it sounds like it would find all quoted `rune`s.

The idea was to have the same naming convention. Before this PR, there was a function `SplitQuoted`, it works like `strings.Split`, but respects quotes. That's why I named this one `FindQuoted`, it finds `rune`, but respects quotes.

I can rename all functions to `FindUnquoted`, `SplitUnquoted` and `SplitAfterUnqioted`, WDYT?

Reply:
I think renaming everything to Unquoted would make sense

Reply:
fixed 🙂